Sources/Afterpay/Views/LogoView.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  LogoView.swift
+//  Afterpay
+//
+//  Created by Scott Antonac on 7/4/2022.
+//  Copyright © 2022 Afterpay. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public class AfterpayLogo: UIView {
+  public var colorScheme: ColorScheme = .static(.blackOnMint) {
+    didSet { updateImage(withTraits: traitCollection) }
+  }
+
+  internal var image: UIImage?
+  private var imageView = UIImageView(frame: .zero)
+
+  internal var minimumWidth: CGFloat = 64
+
+  public var ratio: CGFloat?
+
+  public init(colorScheme: ColorScheme = .static(.blackOnMint)) {
+    self.colorScheme = colorScheme
+
+    super.init(frame: .zero)
+
+    sharedInit()
+  }
+
+  required init?(coder: NSCoder) {
+    super.init(coder: coder)
+
+    sharedInit()
+  }
+
+  internal func sharedInit() {
+    isAccessibilityElement = true
+    updateImage(withTraits: traitCollection)
+
+    let selector = #selector(configurationDidChange)
+    let name: NSNotification.Name = .configurationUpdated
+    notificationCenter.addObserver(self, selector: selector, name: name, object: nil)
+  }
+
+  override public func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+    super.traitCollectionDidChange(previousTraitCollection)
+    updateImage(withTraits: traitCollection)
+  }
+
+  internal func getImageName(brand: String, color: String) -> String {
+    print("is getting: LogoView")
```

Comment:
Is this print for debugging? If so, should it be removed before this is merged?

Reply:
Addressed in c990f90